digitalpy/core/component_management/component_management_facade.py

```diff
@@ -1,6 +1,6 @@
-from Catalog.Data.Domain.FTS_Model.Facade import Facade
+from digitalpy.core.component_management.impl.default_facade import DefaultFacade
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module